Add docs for sharing design and managing access permissions (#231)
layer5io/docs: content/en/meshmap/designer/share-resource/_index.md

---
title: Sharing Designs
description: >
  Share design with other users and use control access to manage design access permissions and visibility.
weight: 3
categories: [Designer]
tags: [designs, collaboration, review]
---

## Overview

> In MeshMap, you can share your designs or design that you have permission to share with other members of your organization and teams and can control over access permissions. This page describes access types for designs and how to effectively use them.

Reviewer: I think the first sentence is too long and as a result ambiguous. Maybe we should break it up into two short sentences.

A suggestion:
“In MeshMap, you can share your designs or designs that you have permission to share with other members of your organization and teams. You also have control over the access permissions for these designs.”

> In MeshMap, you can share your designs or design that you have permission to share with other members of your organization and teams and can control over access permissions. This page describes access types for designs and how to effectively use them.

Reviewer: ```suggestion
In MeshMap, you can share your designs or designs that you have permission to share with other members of your organization and teams. You also have control over access permissions for these designs. This page describes access types for designs and how to effectively use them.
```

> In MeshMap, you can share your designs or design that you have permission to share with other members of your organization and teams and can control over access permissions. This page describes control access permissions for designs and how to effectively use them.

Author: ```suggestion
In MeshMap, you can share your designs or design that you have permission to share with other members of your organization and teams and can control over access permissions. This page describes access types for designs and how to effectively use them.
```

{{< alert title="Note">}}
- You can share and use control access to manage views permissions and visibility also with same methods.
{{< /alert >}}

## Visibility

> Designs and Views have visibility status which defines who can access your designs. These options provide different levels of exposure for your designs:

Reviewer: "Designs and Views have a visibility status which defines who can access them."

> Designs and Views have visibility status which defines who can access your designs. These options provide different levels of exposure for your designs:

Reviewer: ```suggestion
Designs and Views have a visibility status which defines who can access them. These options provide different levels of exposure for your designs:
```

> - **Private:** Designs or Views with visibility status private defines only you, the creator, and the user or team that have access based on granted access permission can view and edit the design. Other users cannot access it unless you explicitly share it with them.

Reviewer: ```suggestion
- **Private:** Designs or Views set to **private** are accessible only to you, the creator, and to users or teams granted access permission. They can view and edit the design. Other users cannot access it unless you explicitly share it with them.
```
